global-cli/index.js

```diff
@@ -67,21 +68,23 @@ if (commands.length === 0) {
 createApp(commands[0], argv.verbose, argv['scripts-version']);
 
 function createApp(name, verbose, version) {
-  if (fs.existsSync(name)) {
-    console.log('The directory `' + name + '` already exists. Aborting.');
+  var root = path.resolve(name);
+  if (!pathExists.sync(name)) {
+    fs.mkdirSync(root);
+  }
```

Comment:
nit: put } before else if. Otherwise skimming through the file we may read as if that's the only condition